test/unit/src/server/launchpad/t_client.js

```diff
@@ -0,0 +1,415 @@
+/* Copyright 2011-2016 Canonical Ltd.  This software is licensed under the
+ * GNU Affero General Public License version 3 (see the file LICENSE).
+ */
+
+import expect, { assert } from 'expect';
+import nock from 'nock';
+
+import { conf } from '../../../../../src/server/helpers/config';
+import {
+  Collection,
+  Entry,
+  normalizeURI,
+  wrapResourceOnSuccess
+} from '../../../../../src/server/launchpad/client';
+import getLaunchpad from '../../../../../src/server/launchpad';
+
+const LP_API_URL = conf.get('LP_API_URL');
+
+describe('normalizeURI', () => {
```

Comment:
It would be best to have one root `describe` per test file. If needed split into several `context` groups for testing different cases.

It goes in line with comment about splitting the lib into several files.
I think that `describe` groups as seen in this test are good candidates to be in separate source files. Or at least in separate test files.